src/main/resources/application.conf

```diff
@@ -0,0 +1,22 @@
+server {
+  host = 0.0.0.0
+  port = 8080
+  ask-timeout = 5s
+}
+
+downloader {
+  max-capacity = 2
```

Comment:
`max-capacity` suggests an upper limit on the number of elements enqueued, in other places `parallelism` is used somehow interchangeably with `max-capacity` which probably would require some standardization.

Reply:
You are correct, it seems that there is no point to have both values so I completely removed one of them!